ui/raidboss/data/04-sb/trial/lakshmi.txt

```diff
@@ -0,0 +1,98 @@
+# Emanation (normal mode)
+# Lakshmi
+
+hideall "--Reset--"
+hideall "--sync--"
+hideall "Alluring Arm"
+
+# -ii 2157 248B 248D
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+0 "Start" sync /Engage!/ window 0,1
+3.9 "Vril" sync /:Dreaming Kshatriya:2482:/ window 3.9,5
+13.0 "Stotram" sync /:Lakshmi:2483:/ window 13,5
+16.1 "Inner Demons" # sync /:Dreaming Kshatriya:258D:/
+19.0 "Aether Drain" sync /:Vril:248D:/
+29.3 "Tail Slap" # sync /:Dreaming Kshatriya:258C:/
+
+# We can't sync to abilities here to make the loop visible,
+# since two adds are present, and the timeline would jerk around.
+60.0 "--sync--" sync /.*/ window 0,30 jump 30
+
+100.0 "--sync--" sync /:Lakshmi:2157:/ window 100,5
+107.1 "Hand Of Grace" sync /:Lakshmi:2486:/
+121.3 "Blissful Arrow" sync /:Lakshmi:2489:/
+121.9 "Blissful Spear" sync /:Lakshmi:248B:/
+132.3 "The Pull Of Light" sync /:Lakshmi:2492:/
+144.5 "Hand Of Beauty" sync /:Lakshmi:2487:/
+158.8 "Blissful Spear" sync /:Lakshmi:2494:/
+165.7 "Stotram" sync /:Lakshmi:2483:/
+167.9 "--vril spawn--"
+169.0 "--untargetable--"
+
+169.9 "Jagadishwari" sync /:Lakshmi:2342:/ window 169.9,30
+186.1 "--sync--" sync /:Lakshmi:2459:/
+189.6 "--stun--"
+190.4 "--sync--" sync /:Lakshmi:245A:/
+191.2 "--sync--" sync /:Lakshmi:248E:/
+206.9 "Alluring Embrace" sync /:Lakshmi:2496:/
+211.3 "--targetable--"
+
+214.5 "Chanchala" sync /:Lakshmi:2484:/ window 214.5,15
+216.7 "--chanchala gain--"
+222.6 "Stotram" sync /:Lakshmi:249E:/
+224.5 "--vril spawn--"
+234.9 "Divine Denial" sync /:Lakshmi:2485:/ window 234.9,30
+234.9 "--vril despawn--"
+247.1 "Hand Of Beauty" sync /:Lakshmi:2487:/
+254.3 "The Pull Of Light" sync /:Lakshmi:2493:/
+261.4 "Blissful Spear" sync /:Lakshmi:2495:/
+270.7 "The Path Of Light" sync /:Lakshmi:24A1:/ window 270.7,30
+284.9 "Hand Of Grace" sync /:Lakshmi:2486:/
+292.1 "The Pall Of Light" sync /:Lakshmi:2491:/
+299.1 "Blissful Arrow" sync /:Lakshmi:248A:/
+299.7 "Blissful Spear" sync /:Lakshmi:248C:/
+305.3 "Stotram" sync /:Lakshmi:249E:/
+307.6 "--vril spawn--"
+320.2 "Divine Denial" sync /:Lakshmi:2485:/ window 30,30
+320.2 "--vril despawn--"
+321.6 "--chanchala lose--"
```

Comment:
The extreme version uses `--Chanchala over--` I don't know if it makes sense to bring that over here as well

Reply:
Maybe we could change both to the more precise `--Chanchala ends--`?

Reply:
I think any of these options are fine, but having both files be consistent would be good.

Reply:
Alright, then I'll make some changes to both to make it maybe a little more sensible. I do think `chanchala over` doesn't really convey what we want, and looking at it more, `chanchala gain` isn't good either.

(At some point I'll need to go through and redo the EX timeline as well, since it's from the days of manual timing. It looks great, but there aren't a lot of syncs.)